secant/Features/Profile/ProfileStore.swift

```diff
@@ -35,9 +35,11 @@ enum ProfileAction: Equatable {
 // MARK: - Environment
 
 struct ProfileEnvironment {
+    typealias ShieldedAddressSource = () -> String?
```

Comment:
We don't have specification for this so first it's probably good to have a conversation where to have such definitions.